src/openrct2/peep/Staff.cpp

```diff
@@ -463,7 +465,7 @@ static uint8_t staff_handyman_direction_to_nearest_litter(Peep* peep)
     int16_t x_diff = litterTile.x - peep->x;
     int16_t y_diff = litterTile.y - peep->y;
 
-    uint8_t nextDirection = 0;
+    Direction nextDirection = 0;
 
     if (abs(x_diff) <= abs(y_diff))
```

Comment:
I feel as though there is a function that does this already. Its used in a number of locations.

Reply:
I've had a look in Location.hpp and Util.h but couldn't see any obvious candidate.

Reply:
Introduced `direction_from_to`.